server/lib/src/schema.rs

```diff
@@ -1763,6 +1765,21 @@ impl<'a> SchemaWriteTransaction<'a> {
                 syntax: SyntaxType::Uint32,
             },
         );
+        self.attributes.insert(
```

Comment:
Attrs in schema.rs are intended to be "critical" to boot the server, and everything else is loaded via migrations. So does this need to be here? Does a "critical" element in the schema depend on this?

Reply:
Weeeelllll it could be once we add a system-wide logo that'd probably be tied into domain_info?